oppia/oppia-android
# Fix part of #1476: Lesson Thumbnail now supports

## Explanation


Fixes part of #1476 

Currently all the thumbnails in the application like TopicThumbnail, StoryThumbnail, ChapterThumbnail and SubtopicThumbnail are getting fetched from `drawable` folder. 

1. This PR adds support for fetching images from oppia-server and as well as drawable in case of test topics.
2. This PR also generalise the `recommendedStoryList` such that it will fetch first exploration from each topic and show it in the app as `recommendedStory`
3. Removed unnecessary details from `proto` files.

## How to test:
Just run the app and you will see that actually there is not change in the images that means the code update works correctly at least for the dummy-test topics. To test this works with server based topics or not, we will need to create a separate PR because that PR will introduce changes in `app` layer too.

## Checklist
